recipes/slda/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+export USE_CYTHON=True
+
+export CC=gcc CXX=g++
```

Comment:
Yeah, we shouldn't need to do that export anymore. It's a fossil from when I had to build a supporting package at the same time (which is now in conda-forge).